docs/src/pages/components/trap-focus/BasicTrapFocus.js

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import TrapFocus from '@material-ui/core/Unstable_TrapFocus';
+
+export default function BasicTrapFocus() {
+  const [open, setOpen] = React.useState(false);
+  return (
+    <div>
+      <button type="button" onClick={() => setOpen(true)}>
+        Open
+      </button>
+      <br />
+      {open && (
+        <TrapFocus open isEnabled={() => true} getDoc={() => document}>
+          <div tabIndex={-1}>
+            <h3>Quick form</h3>
+            <input aria-label="first name" placeholder="First name" />
```

Comment:
Can we use a more basic setup that doesn't use problematic attributes such as `placeholders`?
`<label>first name: <input type="text" /></label>` would work just as well. (added `type` for explicitness since these are docs.